### ISSUE TITLE
When zooming out, detailed imagery tiles should not disappear if less detailed tiles are not available.

### ISSUE THREAD

**Issue by crubier** [iTowns/itowns#1223]
When zooming out of a view where all imagery tiles are loaded, the detailed tiles disappear, even before the less detailed tiles are loaded. 

The problem is seen happening repeatedly on this screen capture:

![Screen Recording 2019-09-07 at 11 01 38 mov](https://user-images.githubusercontent.com/2954572/64473462-00618480-d15f-11e9-9773-4d2222d4f83d.gif)

If detailed tiles are available but less detailed tiles are not available, then the detailed tiles should be visible even if they are too detailed for the current zoom level. Temporarily showing a bit too much detail is better than temporarily showing nothing/very low level of detail.

We are considering moving from cesium to itowns, but this (and a few other bugs) are preventing us from doing so.

**Comment by gchoqueux** [comment]
Yes, unfortunately, it is a known behaviour.  We want to improve the rendering, we could plan it in the next 3 to 6 months.
